menu-bridges

```diff
@@ -271,31 +259,48 @@ if [ $exitstatus = 0 ]; then
       read -n 1 -s -r -p "Press any key to continue"
 
 # Add bridges
-    elif [ $CHOICE = 4 ]; then
+    elif [ $CHOICE = 3 ]; then
       sudo bash bridges_add_old $MODE_BRIDGES
```

Comment:
Should an activated Meek-Azure/Snowflake block access to that function?

Reply:
Even if is a note to self, I think should block access to all OBFS4 functions. Good catch, didn't think of it before.